app/allocations/views/list.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/list-view.njk" %}
+
+{% block headerGridColumnClass %}govuk-grid-column-full{% endblock %}
+
+{% block headerActions %}
+  {% if canAccess('move:create') %}
+    {{ govukButton({
+      isStartButton: true,
+      text: t("actions::create_move"),
+      href: "/allocations/new",
+      classes: "govuk-!-margin-top-2 app-print--hide"
+    }) }}
+  {% endif %}
+  {{ appFilter({
+    items: allocationTypeNavigation
+  }) }}
+{% endblock %}
+
+{% block pageContent %}
+{% if allocations.length %}
+  {{ govukTable({
+    rows: allocations,
+    head: allocationsHeads
+  }) }}
+{% else %}
+    {{ appMessage({
+      classes: "app-message--muted govuk-!-margin-top-7",
+      allowDismiss: false,
+      content: {
+        html: t("moves::dashboard.single_moves_no_results")
```

Comment:
Is this using the correct key? Looks like its using content from single requests which might differ.

Reply:
Good point, I shouldn't recycle tokens! Shall amend.